lnbits/core/views/generic.py

```diff
@@ -217,7 +216,7 @@ async def pay():
 
 
 @core_html_routes.get("/deletewallet", response_class=RedirectResponse)
-async def deletewallet(request: Request, wal: str = Query(...), usr: str = Query(...)):  # type: ignore
```

Comment:
Can you delete the argument without breaking things?

Reply:
same as in the other pr